src/common/StatusBar/index.tsx

```diff
@@ -0,0 +1,14 @@
+"use client";
+import { useUser } from "@/hooks/useGetLoggedinUser";
+import React from "react";
+export const AlertBar = () => {
+  const loggedInUser = useUser();
+  if (loggedInUser?.temporary_password) {
```

Comment:
is this condition correct i think if temporary password is present then we should show alertbar otherwise we will return nothing instead of returning a empty react fragment

Reply:
Yes, the condition is correct because we have default value is boolean `TRUE`, and I am currently returning React.Fragment. If I don't return anything, it results in undefined, which is not valid JSX. We can improve this by returning null instead of React.Fragment.